### DISCUSSION TITLE
ERROR: linux-aspeed-6.1.29+gitAUTOINC+1ac953dfce-r0 do_fetch: Fetcher failure: Fetch command export

### DISCUSSION THREAD

**Discussion by wangliuusa** [community/community#56447] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Sir/Madam,

Could you please help me in fixing the following build error ?

Sincerely,
Liu Wang
--------------------------------------------------------------------------------------------------
liuw@MININT-9CCHGDM:~/openbmc/build/x11spi$ bitbake obmc-phosphor-image
WARNING: Host distribution "ubuntu-22.04" has not been validated with this version of the build system; you may possibly experience unexpected failures. It is recommended that you use a tested distribution.
Loading cache: 100% |#############################################################################################################################################################################| Time: 0:00:00Loaded 4421 entries from dependency cache.
Parsing recipes: 100% |###########################################################################################################################################################################| Time: 0:00:00Parsing of 2725 .bb files complete (2724 cached, 1 parsed). 4422 targets, 575 skipped, 0 masked, 0 errors.
WARNING: No bb files in default matched BBFILE_PATTERN_x11spi ''
NOTE: Resolving any missing task queue dependencies

Build Configuration:
BB_VERSION           = "2.4.0"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "ubuntu-22.04"
TARGET_SYS           = "arm-openbmc-linux-gnueabi"
MACHINE              = "x11spi"
DISTRO               = "openbmc-phosphor"
DISTRO_VERSION       = "nodistro.0"
TUNE_FEATURES        = "arm thumb arm1176jzs"
TARGET_FPU           = "soft"
meta
meta-poky
meta-oe
meta-networking
meta-python
meta-phosphor
meta-aspeed
meta-supermicro
meta-x11spi          = "master:4b666412be40270df70c4bce2c9677c629dfb31b"

Initialising tasks: 100% |########################################################################################################################################################################| Time: 0:00:02Sstate summary: Wanted 1184 Local 0 Mirrors 0 Missed 1184 Current 1279 (0% match, 51% complete)
NOTE: Executing Tasks
WARNING: linux-aspeed-6.1.29+gitAUTOINC+1ac953dfce-r0 do_fetch: Failed to fetch URL git://github.com/openbmc/linux;protocol=https;branch=dev-6.1, attempting MIRRORS if available
ERROR: linux-aspeed-6.1.29+gitAUTOINC+1ac953dfce-r0 do_fetch: Fetcher failure: Fetch command export PSEUDO_DISABLED=1; export PATH="/home/liuw/openbmc/scripts:/home/liuw/openbmc/build/x11spi/tmp/work/x11spi-openbmc-linux-gnueabi/linux-aspeed/6.1.29+gitAUTOINC+1ac953dfce-r0/recipe-sysroot-native/usr/bin/arm-openbmc-linux-gnueabi:/home/liuw/openbmc/build/x11spi/tmp/work/x11spi-openbmc-linux-gnueabi/linux-aspeed/6.1.29+gitAUTOINC+1ac953dfce-r0/recipe-sysroot/usr/bin/crossscripts:/home/liuw/openbmc/build/x11spi/tmp/work/x11spi-openbmc-linux-gnueabi/linux-aspeed/6.1.29+gitAUTOINC+1ac953dfce-r0/recipe-sysroot-native/usr/sbin:/home/liuw/openbmc/build/x11spi/tmp/work/x11spi-openbmc-linux-gnueabi/linux-aspeed/6.1.29+gitAUTOINC+1ac953dfce-r0/recipe-sysroot-native/usr/bin:/home/liuw/openbmc/build/x11spi/tmp/work/x11spi-openbmc-linux-gnueabi/linux-aspeed/6.1.29+gitAUTOINC+1ac953dfce-r0/recipe-sysroot-native/sbin:/home/liuw/openbmc/build/x11spi/tmp/work/x11spi-openbmc-linux-gnueabi/linux-aspeed/6.1.29+gitAUTOINC+1ac953dfce-r0/recipe-sysroot-native/bin:/home/liuw/openbmc/poky/bitbake/bin:/home/liuw/openbmc/build/x11spi/tmp/hosttools"; export HOME="/home/liuw"; LANG=C git -c gc.autoDetach=false -c core.pager=cat clone --bare --mirror https://github.com/openbmc/linux /home/liuw/openbmc/build/x11spi/downloads/git2/github.com.openbmc.linux --progress failed with exit code 128, no output
ERROR: linux-aspeed-6.1.29+gitAUTOINC+1ac953dfce-r0 do_fetch: Bitbake Fetcher Error: FetchError('Unable to fetch URL from any source.', 'git://github.com/openbmc/linux;protocol=https;branch=dev-6.1')
ERROR: Logfile of failure stored in: /home/liuw/openbmc/build/x11spi/tmp/work/x11spi-openbmc-linux-gnueabi/linux-aspeed/6.1.29+gitAUTOINC+1ac953dfce-r0/temp/log.do_fetch.1895
ERROR: Task (/home/liuw/openbmc/meta-aspeed/recipes-kernel/linux/linux-aspeed_git.bb:do_fetch) failed with exit code '1'
NOTE: Tasks Summary: Attempted 3330 tasks of which 2885 didn't need to be rerun and 1 failed.

Summary: 1 task failed:
  /home/liuw/openbmc/meta-aspeed/recipes-kernel/linux/linux-aspeed_git.bb:do_fetch
Summary: There were 3 WARNING messages.
Summary: There were 2 ERROR messages, returning a non-zero exit code.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by mehmet1990xx** [comment]
Codlar değişmiş ti. Oynanmisti ....
 Androit 13 sürüm telafona  yükleme  yapıldı çözüm odaklı  sorun girmer

**Comment by mehmet1990xx** [comment]
Deneyim farlı  kodlar yada  seri  kullanım oldu

**Comment by mehmet1990xx** [comment]
X  ilk defa github tanidim.  O gün  telefonumda  neleri yapabileceğimi  düşündüm. İlk tanisma  yaşanması gereken. Github ile bir yolculuk  birlikte...

**Comment by mehmet1990xx** [comment]
Özel  1   elimde  var androit  telefon.   kime güvenir vereceksin.  Codlari  telefonumda genel  acilimlarda böyle ufak ahatalari  çözümlemek deneyimi olanların yardımlarıyla   githubmcxx  sorunlarının  çözümü birlikteligimizin devamı kategori

**Comment by mehmet1990xx** [comment]
Kategori  deneyelim  keman mn (0)

**Comment by mehmet1990xx** [comment]
Githubmcxx  yanan atesinden..
küllerinden yeniden  dogar

**Reply by wangliuusa** [reply]
teşekkür

**Comment by wangliuusa** [comment]
Sir/Madam,

Could anyone please recommend a successful build setup for MACHINE="x11spi ?"

Sincerely,
Liu Wang

**Comment by wangliuusa** [comment]
Close it for now to transfer to an Aspeed2600 EVB board.
Thank you anyway.

Sincerely,
Liu
